### PR TITLE
ELEMENTS-703: Add "zoom to real size" functionality

### PR DESCRIPTION
**Pull request by nmpcunha**

### CODE REVIEW

File: viewers/nuxeo-image-viewer.html [hunk 1]
```diff
@@ -189,12 +207,38 @@
...
           }
         }
 
+        _verifyZoomRatio(data){
+          if(this._el){
+            // Cropper.js not always return the same number of decimal places and rounding.
+            // In order to ensure our calculations, we will use 5 decimal places.
+            const decimalPlaces = 5;
+
+            if(this._getOriginalZoomRatio().toFixed(decimalPlaces) === data.ratio.toFixed(decimalPlaces)){
```
Comment (afjusto) [inline viewers/nuxeo-image-viewer.html]: `this._fitToViewer = this._getOriginalZoomRatio().toFixed(decimalPlaces) === data.ratio.toFixed(decimalPlaces);`?

File: viewers/nuxeo-image-viewer.html [hunk 2]
```diff
@@ -189,12 +207,38 @@
           }
         }
 
+        _getFitIcon(fitToViewer) {
+          return fitToViewer ? 'nuxeo:fit-to-real-size' : 'nuxeo:fit-to-viewer';
+        }
+
+        _getOriginalZoomRatio(){
+          if(this._el){
```
Comment (afjusto) [inline viewers/nuxeo-image-viewer.html]: format this (missing some space) and other places below too

File: viewers/nuxeo-image-viewer.html [hunk 3]
```diff
@@ -139,6 +142,15 @@
...
               value: false,
             },
+
+            /**
+             * If true, the element will show the image in order to fit the viewer.
+             * If false, the element will show the image in real size. 
+             */
+            _fitToViewer: {
+              type: Boolean,
+              value: true,
```
Comment (afjusto) [inline viewers/nuxeo-image-viewer.html]: we usually don't set the default value in booleans as `true` (see https://www.polymer-project.org/2.0/docs/devguide/properties#configuring-boolean-properties for details), but in this case I think there's no problem (given the explanation in the polymer docs).

File: viewers/nuxeo-image-viewer.html [hunk 4]
```diff
@@ -164,6 +176,9 @@
               toggleDragModeOnDblclick: false,
               viewMode: 1,
               zoomOnWheel: this.zoomOnWheel,
+              zoom: (data) => {
```
Comment (afjusto) [inline viewers/nuxeo-image-viewer.html]: I guess you don't need the curly braces? like `zoom: (data) => this._verifyZoomRatio(data),`

**Comment by afjusto** [comment]
Lint is complaining about a couple of stuff, don't forget to look at https://qa.nuxeo.org/jenkins/job/Other/job/nuxeo-ui-elements-pr-check/348/console

**Comment by Gabez0r** [comment]
👍

**Comment by afjusto** [comment]
👍

**Comment by Gabez0r** [comment]
👍

**Pull request merged**